app/services/food_service.rb

```diff
@@ -0,0 +1,20 @@
+class FoodService
+  class << self
+    def call_for_a_food(food)
+      response = connection.get("/v1/foods/search?query=#{food}")
+      binding.pry
+      parse_data(response)
+    end
+
+  private
+    def connection
+      Faraday.new(url: "https://api.nal.usda.gov/fdc") do |faraday|
```

Comment:
I believe the reason your connection is failing is because your connection includes `/fdc` which is technically part of the path (so it can go in `connection.get('/fdc/v1/foods . . . '`). The general rule is that your Faraday connection should include the base URL only, so it shouldn't generally include any path information (with a `/`) but I imagine this was a very frustrating error to try to debug!

Reply:
The rest of your setup looks solid!